qa/tasks/cephfs/test_volumes.py

```diff
@@ -4410,7 +4410,7 @@ def test_subvolume_snapshot_info(self):
         tests the 'fs subvolume snapshot info' command
         """
 
-        snap_md = ["created_at", "data_pool", "has_pending_clones"]
+        snap_md = ["bytes_quota", "created_at", "data_pool", "has_pending_clones"]
```

Comment:
Can we add the following two tests comparing the contents of the `bytes_quota` ?

1. Create a subvolume with no quota set, snap it and validate the `bytes_quota` to `infinite`
2. Create a subvolume with quota set, snap it and validate the `bytes_quota` to whatever was set.

Reply:
Addressed